packages/webapp/src/encryption/components/reenter-password-form.tsx

```diff
@@ -59,13 +64,23 @@ export const ReenterPasswordForm = ({
                         disabled={isLoading}
                         onChange={onChangeFields}
                     />
-                    <Link onClick={onForgotPassword}>Forgot Password?</Link>
+                    <Button
+                        type="link"
+                        onClick={onForgotPassword}
+                        className="pl-0 pr-0"
+                    >
+                        I forgot my password
+                    </Button>
                 </Form.LabeledSet>
                 <div className="flex space-x-3">
                     <Button type="neutral" onClick={onCancel}>
                         Cancel
                     </Button>
-                    <Button isSubmit disabled={!fields.password || isLoading}>
+                    <Button
+                        isSubmit
+                        disabled={!fields.password || isLoading}
+                        isLoading={isLoading}
```

Comment:
**future-enhancement (not blocking for this pr):** I feel like the isLoading flag should automatically disable the button in the button component. I can't think of a case where a process is loading but we want the button to be able to be pressed again... I think that would remove some redundancy in our code like this:
`<Button isSubmit disabled={!fields.password} isLoading={isLoading} />`

Reply:
Yeah, that's crossed my mind too. It makes sense.